src/pages/resources/recipes/xml/index.md

```diff
@@ -5,8 +5,7 @@
 Once you set the XML import preferences the way you want them, you can import an XML file, as shown in the following script fragment (from the ImportXML tutorial script):
```

Comment:
Not sure what this line is referring to... Can you add the details of 'XML import preferences'?
What's "from the ImportXML tutorial script"?